changelogs/fragments/1916-add-version-sort-filter.yml

```diff
@@ -0,0 +1,6 @@
+add plugin.filter:
+  - name: version_sort
+    description: sort a list according to version order instead of pure alphabetical one
+add plugin.test:
+  - name: filter_version_sort
+    description: make sure that the version_sort filter works properly
```

Comment:
```suggestion
```
You are not adding a test plugin, but a test. That doesn't need a changelog fragment :)

Reply:
Need to learn what a test plug-in is, but corrected.